iohub/ngff/nodes.py

```diff
@@ -1010,6 +1010,80 @@ def get_axis_index(self, axis_name: str) -> int:
         """
         return self.axis_names.index(axis_name.lower())
 
+    def get_effective_scale(
```

Comment:
The `scale` property can now use this method instead of implementing its own logic.